Tests/components/oculus-go-controls.test.js

```diff
@@ -0,0 +1,263 @@
+/* global assert, process, setup, sinon, suite, test */
+var entityFactory = require('../helpers').entityFactory;
```

Comment:
the path for this file should be `tests/components/oculus-go-controls.test.js` with lowercase `tests` instead of `Tests`